data/json/items/gun/762.json

```diff
@@ -243,7 +243,7 @@
     "looks_like": "ar15",
     "type": "GUN",
     "name": { "str": "vz. 58 combat rifle" },
-    "description": "A Cold War era Czechoslovakian assault rifle, likely constructed stateside from a pre-1986 imported parts kit or brought back from a third world war zone.  Externally similar to the AK family of rifles, this weapon nevertheless only shares the former’s 7.62x39mm chambering.  Featuring no interchangeable parts with its Russian doppelganger, this firearm makes use of a completely different internal operating system, is far more svelte than its other Warsaw Pact counterparts, and feeds from proprietary box magazines.",
+    "description": "A Cold War-era Czechoslovak assault rifle, likely constructed stateside from a pre-1986 imported parts kit or brought back from a third world war zone.  Externally similar to the AK family of rifles, this weapon nevertheless only shares the former’s 7.62x39mm chambering.  Featuring no interchangeable parts with its Russian doppelganger, this firearm makes use of a completely different internal operating system, is far more svelte than its other Warsaw Pact counterparts, and feeds from proprietary box magazines.",
```

Comment:
If you're going to shorten the name, why not go all the way down to Czech?

Reply:
I shortened it because Czechoslovakian is incorrect. To shorten it to Czech would imply it's from the modern day Czech Republic and not Czechoslovakia.